### PR TITLE
Fix #7448: added milliseconds to datepicker

### LINKED ISSUE #7448: DatePicker: Add support for fractions of a second
To allow migrating some use cases from Calendar component, which is deprecated now, support for entering/selecting milliseconds should be added to DatePicker. It's a functionality that is present in Calendar, but not in DatePicker. While doing this, you may consider adding support for micro- or nanoseconds (any precision from 0 to 9).

### PR DESCRIPTION
**Pull request by wal-jan**
Added milliseconds to DatePicker (but not micros nor nonos since JS Date only supports milliseconds). Also added some integration tests and augmented DatePicker showcases.

### CODE REVIEW

**Comment by melloware** [comment]
This looks like it was a lot of work!   Nice PR we will all review it.

File: primefaces/src/main/resources/META-INF/resources/primefaces/core/core.js [hunk 1]
```diff
@@ -1344,6 +1344,7 @@
             hourText: 'Hour',
             minuteText: 'Minute',
             secondText: 'Second',
+            millisecondText: 'Millisecond',
```
Comment (jepsar) [inline primefaces/src/main/resources/META-INF/resources/primefaces/core/core.js:1347]: We should add this to the locale files as well. @melloware as a separate PR?

Comment (melloware) [reply primefaces/src/main/resources/META-INF/resources/primefaces/core/core.js:1347]: I am fine with separate PR or if @wal-jan wants to add them to the local files in this PR.

Comment (wal-jan) [reply primefaces/src/main/resources/META-INF/resources/primefaces/core/core.js:1347]: @jepsar , @melloware: I'll add localized texts to PR (Google translate hopefully will do the translations)

Comment (melloware) [reply primefaces/src/main/resources/META-INF/resources/primefaces/core/core.js:1347]: Nice yeah at least that will be a good start and people can always complain and we can fix if its not perfect. Much appreciated.

**Comment by wal-jan** [comment]
Done adding localized texts (see [068b70c](https://github.com/primefaces/primefaces/pull/7490/commits/068b70c71b7d2bdbd15a2f17d198b36bd0b1fcfe)) Do I have to issue a new PR or ist this change sufficient?

**Comment by melloware** [comment]
Nope this PR is good. Thanks for updating those locales!

**Review by melloware — APPROVED** [review]

**Comment by tandraschko** [comment]
great work and PR - thanks!

**Pull request merged**